GUIDELINES.md

```diff
@@ -0,0 +1,70 @@
+Spider writing guidelines
+=========================
+
+Before start working on a spider, read carefully the following guidelines that will make the project more consistent and your PR more probable to be accepted.
+
+Reading code of existing spiders will also help you to understand how to develop.
+
+Spider class
+------------
+
+- All spiders must inherit from `BaseGazetteSpider`
+- Class name must follow the pattern `<State><CityName>Spider` (for example `SpSaoPauloSpider` or `MtFelizNatalSpider`)
+- Spider `name` must follow the pattern `<state>_<city_name>` (for example `sp_sao_paulo` or `mt_feliz_natal`)
+- Add `TERRITORY_ID` attribute with the code found in `territories.csv` file
```

Comment:
```suggestion
- Add `TERRITORY_ID` attribute with the code found in `data/territories.csv` file
```

Cool! I love the built-in search provided by Github for csv files. I think we mentioned a Wikipedia page somewhere... maybe we should update it to this file as well.